packages/cra-template-seed/template/src/auth/Components/Validator.js

```diff
@@ -0,0 +1,12 @@
+import { useHasAccess } from '../../hooks/useAuthority';
+
+const ComponentValidator = ({ allowedAuthorities, authorityKey, children }) => {
```

Comment:
can we come up with a better name here? Component Validator sounds like it is checking some static stuff in an component. Maybe something like `AuthGate`? or any other name :)